<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialoginsert</class>
 <widget class="QDialog" name="dialoginsert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresources.qrc">
    <normaloff>:/png/png/insert.png</normaloff>:/png/png/insert.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTimeEdit" name="starttimeEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="maximumTime">
    <time>
     <hour>9</hour>
     <minute>59</minute>
     <second>59</second>
    </time>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::HourSection</enum>
   </property>
   <property name="displayFormat">
    <string>HH:mm:ss.zzz</string>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
   <property name="time">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="text_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>End time:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QTimeEdit" name="endtimeEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumTime">
    <time>
     <hour>9</hour>
     <minute>59</minute>
     <second>59</second>
    </time>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::HourSection</enum>
   </property>
   <property name="displayFormat">
    <string>HH:mm:ss.zzz</string>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Text:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
